helpers/rcon.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"fmt"
+	"github.com/gorcon/rcon"
+	"strconv"
+)
+
+func (c *HelpersController) SendRconCommand(command string, ip string, port int, password string) (string, error) {
+	conn, err := rcon.Dial(ip+":"+strconv.Itoa(port), password)
+	if err != nil {
+		return "", fmt.Errorf("failed connectting to rcon server: %v", err)
+	}
+	defer conn.Close()
```

Comment:
Should this not already be closed if the connection failed?

Reply:
its a defer, it just runs at the end if it is open